src/components/TodoForm/TodoForm.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import './TodoForm.css';
+import PropTypes from 'prop-types';
+
+export class TodoForm extends React.Component {
+  state = {
+    todo: '',
+    userId: 0,
+    todoError: false,
+    userIdError: false,
+  }
+
+  onSubmitHandler = (event) => {
+    event.preventDefault();
+
+    this.setState(({ todo, userId }) => ({
+      todoError: !todo,
+      userIdError: !userId,
+    }));
+
+    const { userId, todo } = this.state;
+
+    if (!userId) {
+      return;
+    }
+
+    if (!todo) {
+      return;
+    }
+
+    this.props.addTodo(this.state.todo, +this.state.userId);
+    this.setState(
+      {
+        todo: '',
+        userId: 0,
+      },
+    );
+  }
+
+  todoHandler = (event) => {
+    this.setState({
+      todoError: !event.target.value,
+      todo: event.target.value,
```

Comment:
We can add an empty todo (using spaces)

![empty](https://user-images.githubusercontent.com/66570142/119941849-cd49cf00-bf99-11eb-8235-1bda37c08456.png)